onprc_ehr/resources/queries/onprc_ehr/RandalData_MHCData.sql

```diff
@@ -8,6 +13,10 @@ SELECT d.rowid,
        d.assaytype,
        d.container,
        d.totalTests
+<<<<<<< HEAD
```

Comment:
Does this line marked as a sql comment like adding "---" three  hyphen symbol preceding your text?  If not, then this will show as an error as sql statement.